forge/routes/ui/index.js

```diff
@@ -22,14 +22,40 @@ module.exports = async function (app) {
     const frontendAssetsDir = path.join(__dirname, '../../../frontend/dist/')
 
     /**
-     * Inject additional script imports for index.html that are dependant upon flowforge.yml
+     * Inject Analytics Tools
+     * feConfig - the 'frontend' portion of our flowforge.yml
      */
-    async function injectPlausible (domain, extension) {
+    async function injnectAnalytics (feConfig) {
```

Comment:
Still has typo in name

```suggestion
    async function injectAnalytics (feConfig) {
```